010_experiments/010_experiments.tex

```diff
@@ -1,8 +1,388 @@
+And by making it harder to survive in the tower, the chances of reaching a stable system are also decreased when disruptive agents are added, the only system to stabilise with 10 food per agent was the one with no disruptive agents, all systems were stable with a tower of 20 food per agent.
+Interestingly
+\subsection{Conclusion}
+\label{subsec: E1-Conclusion}
+This experiment shows the importance of treaties and communication for the survival of the agents in the tower.And so this shows that self organisation is a key factor in the ability of the tower to reach a stable state.
+
+\section{Experiment 2}
+\label{subsec: Experiment 2}
+\subsection{Aims}
+\label{subsec: E2-Aims}
+The aim of this experiment is to very the number of ticks per floor and reshuffle period and observe the effects of this on tower stability and ability to self organise.
+\subsection{Results and Discussion}
+\label{subsec: E2-Results and Discussion}
+
+\begin{figure}[H] %Number of Deaths As Shuffle Period Decreases
+    \centering
+    \begin{minipage}{0.8\textwidth}
+        \begin{tikzpicture}
+            \begin{axis}[
+                width=\textwidth,
+                axis y line*=left,
+                ymin=0,
+                xmin=0,
+                xlabel=Shuffle Period (no. of days),
+                ylabel=Total Deaths,
+                ymajorgrids=true,
+                xmajorgrids=true,
+                grid style=dashed,
+                legend pos=south east
+            ]
+            \addplot[mark=*,blue]
+                coordinates{
+                    (14, 253)
+                    (7, 220)
+                    (2, 188)
+                }; \label{leg:reshuffle-ticks-per-floor1}
+            \addplot[mark=*,red]
+                coordinates{
+                    (14, 229)
+                    (7, 228)
+                    (2, 177)
+                }; \label{leg:reshuffle-ticks-per-floor10}
+            \addlegendimage{/pgfplots/refstyle=leg:reshuffle-ticks-per-floor1}
+            \addlegendentry{1 Tick per Floor}
+            \addlegendimage{/pgfplots/refstyle=leg:reshuffle-ticks-per-floor10}\addlegendentry{10 Ticks per Floor}
+            \end{axis}
+        \end{tikzpicture}
+    \end{minipage}
+    \caption{Total Deaths in the Tower as Reshuffle Period Increases}
+    \label{fig:Number-of-Deaths-As-Shuffle-Period-Decreases}
+\end{figure}
+
+\Cref{fig:Number-of-Deaths-As-Shuffle-Period-Decreases} investigates the correlation between the global deaths in the tower and the time between reshuffle periods, again using the initialisation defined in \ToDo{add a table for init} 
```

Comment:
Just want to note the ToDo here, not sure if still needed?

Reply:
this was removed in a newer commit